### DISCUSSION TITLE
Welcome to ORCA Discussions!

### DISCUSSION THREAD

**Discussion by cammurray** [cammurray/orca#198] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by areyes21** [comment]
Hi cammurray

How can I calculate the CHI percentage from Output JSON? 
<img width="941" alt="image" src="https://github.com/cammurray/orca/assets/37639639/dc5e1861-3829-47ca-95b0-25426945778b">
